scrapy/utils/reactor.py

```diff
@@ -49,13 +49,18 @@ def __call__(self):
         return self._func(*self._a, **self._kw)
 
 
-def install_reactor(reactor_path):
+def install_reactor(reactor_path, event_loop_path=None):
     """Installs the :mod:`~twisted.internet.reactor` with the specified
-    import path."""
+    import path. Also installs the asyncio event loop with the specified import
+    path if the asyncio reactor is enabled"""
     reactor_class = load_object(reactor_path)
     if reactor_class is asyncioreactor.AsyncioSelectorReactor:
         with suppress(error.ReactorAlreadyInstalledError):
-            asyncioreactor.install(asyncio.get_event_loop())
+            if event_loop_path is not None:
+                event_loop_class = load_object(event_loop_path)
+                event_loop = event_loop_class()
+                asyncio.set_event_loop(event_loop)
+            asyncioreactor.install()
```

Comment:
Wait, my bad. It looks like calling `asyncioreactor.install()` is necessary in all cases. Please, feel free to go to your previous implementation. Sorry :disappointed:

Reply:
I think that the current implementation works fine. Once we set the custom event loop in asyncio , it automatically becomes the default loop and `asyncioreactor` installs it. We don't really need to supply any parameter to `asyncioreactor.install()`.

Reply:
Interesting. However, maybe we should avoid calling `asyncio.set_event_loop()`. Changing the default loop globally may cause unexpected issues in projects that execute another asyncio loop after the Scrapy-based crawling finishes.

Reply:
I have changed reactor.py and log.py. Now the event loop is only set in the `asyncioreactor`. The global asyncio event loop is no longer changed.